doc/Language/modules.pod6

```diff
@@ -524,6 +524,65 @@ The directories in the C<RAKULIB> path will be searched for modules when
 Raku C<need>s, C<use>s or C<require>s them. Directories that start with a
 dot are ignored and symlinks are followed.
 
+=head1 Testing modules
+
+Once a module (for illustration C<MyMainModule>) has been written, 
+the basic test (eg., in t/01-sanity.t) it needs to pass is 
+
+    use-ok 'MyMainModule';
+    
+There is an important feature to B<Raku> modules that are distributed, namely
+the presence of a C<META6.json> file in the root of the directory. See below
+for more about C<META6.json>. 
+
+When testing, it is possible to use C<prove6>, the B<Raku> version of C<prove>, eg.
+
+    prove6 -I.
+    
+This should be run in the root directory of the Module, and it will run all the tests in 
+the C<t/> sub-directory. The parameter B<-I.>, rather than B<-Ilib>,
+is significant because then the compiler will inspect the C<META6.json> file in the root
+directory of C<MyMainModule>. 
```

Comment:
Um. Not totally sure about this. I know `zef` does use it, but do you know `prove6` does that? What for? What does it have to do with the search path to find modules?